ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/freemarker/tasks/main.yml

```diff
@@ -21,19 +21,34 @@
     dest: /tmp/
     remote_src: yes
     mode: 0755
-  when: not freemarker.stat.exists
+  when: not freemarker.stat.exists and ansible_distribution != "Solaris"
+  tags: [freemarker, adoptopenjdk]
+
+- name: Download and extract freemarker.jar (Solaris)
```

Comment:
Why have we gone for `unarchive` here with `validate_certs: false` when we've used `wget` elsewhere?

Reply:
I've used get_url wherever possible. It seems to be down to the TLS version that each server accepts

Reply:
Hmmm is it TLS rather than the CA certs? (the latter should be replaceable)